packages/axios-extension/test/abap/ui5-abap-repository-service.test.ts

```diff
@@ -109,7 +113,7 @@ describe('Ui5AbapRepositoryService', () => {
                     `${Ui5AbapRepositoryService.PATH}/Repositories(%27${validApp}%27)?${updateParams}`,
                     (body) => body.indexOf(archive.toString('base64')) !== -1
                 )
-                .replyWithError('Deployment failed');
+                .reply(401, 'Deployment failed');
```

Comment:
What is the reason for this change?

Reply:
This actually provides the AxiosError object populated the error.response.data is populdated with `Deployment failed` which is required to cover some of the logging changes made.